webapps/waterfall-chart/backend.py

```diff
@@ -0,0 +1,65 @@
+import dataiku
+from flask import request
+import pandas as pd
+import numpy as np
+import json
+import traceback
+import logging
+logger = logging.getLogger(__name__)
+
+
+def build_complete_df(df, category_column, value_column, max_displayed_values, group_others):
+    try:
```

Comment:
A python commment could be helpful for future developpers. Explain shortly what the method does, especially the things not easily readable by a human.